heka/meta/heka.yml

```diff
@@ -336,7 +336,8 @@ aggregator:
           threshold: 50
           function: percent
     - status: okay
-    # A policy which is typically used for stateless clusters
+    # A policy that is typically used to derive cluster status based
+    # on the up or down status of its members.
```

Comment:
What happens when we have three members and when member1 is "okay", member2 is "warning" and member3 is "warning"? It seems to me that the policy will emit "unknown" in that case, which does not sound correct to me.